catalogue/webapp/components/Calendar/calendar-utils.ts

```diff
@@ -38,6 +38,11 @@ export function countDaysInMonth(d: Date): number {
   return new Date(d.getFullYear(), d.getMonth() + 1, 0).getDate();
 }
 
+/** Returns a list of the dates in a month.
```

Comment:
Since we're not using the Calendar component (it was superseded by the Select day picker), should we remove it altogether (at least for now), rather than maintaining it just in case?

Reply:
That sounds like a very sensible idea!